README.md

```diff
@@ -78,6 +78,20 @@ You can specify a custom toggle class by adding the `data-toggle-class` attribut
 
 In this example the `toggled` class will be applied to the target element (rather than the default `is-hidden` class).
 
+### Toggle target element, and hide all other elements in a group
+
+Rather than individually listing which elements you would like to hide, ie `data-toggle-target="hide:one hide:two"`, you 
+can apply attr `data-toggle-group-target="name"` to the toggle button, and attr `data-toggle-group="name"` to each 
```

Comment:
I think removing the words "attr" in this sentence makes it read a little better?